### PR TITLE
eggnogmapper eggnog.db

### PR DESCRIPTION
**Pull request by vagkaratzas**

### CODE REVIEW

File: data/delete_me/eggnogmapper/eggnog.db
[the remark this answers is not in the document]
Comment (vagkaratzas) [reply data/delete_me/eggnogmapper/eggnog.db:1]: This is an sqlite3 database that is only used by eggnogmapper (and is mandatory) that can't be re-used by any other module. This specific file is a test version provided in the eggnog-mapper repo (https://github.com/eggnogdb/eggnog-mapper/blob/master/tests/fixtures/eggnog.db). According to the nf-core guidelines (https://nf-co.re/docs/contributing/test_data_guidelines#adding-new-test-data) I think that the delete_me folder is the most suited place to put this.

[the remark this answers is not in the document]
Comment (SPPearce) [reply data/delete_me/eggnogmapper/eggnog.db:1]: This seems rather large at 18.7Mb, do you need all of it for testing?

[the remark this answers is not in the document]
Comment (vagkaratzas) [reply data/delete_me/eggnogmapper/eggnog.db:1]: I manually deleted most of the entries in the sqlite db and kept only 2 rows. The one I renamed to ENSSASP00005000002.1 in order to test with the existing genomics/sarscov2/genome/proteome.fasta. The file is now ~60kB.

File: README.md [hunk 1]
```diff
@@ -84,17 +84,20 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
   - genome
     - 'genome.fasta': MT192765.1 genome including (GCA_011545545.1_ASM1154554v1)
     - 'genome.fasta.gz': bgzipped version of 'genome.fasta'
+    - 'genome.fasta.fai': fasta index for 'genome.fasta'
     - 'genome.fasta.txt.gz': gzipped version of 'genome.fasta' in tabular text format
     - 'genome.fasta.txt.zst': zstd-compressed version of 'genome.fasta' in tabular text format
     - 'genome.dict': GATK dict for 'genome.fasta'
-    - 'genome_strtablefile.zip': An StrTableFile zip folder for 'genome.fasta'
```
[the remark this answers is not in the document]
Comment (vagkaratzas) [reply README.md:90]: Move under homo_sapiens/genome, where it should based on the test_data.config.

**Review by SPPearce — APPROVED** [review]

**Pull request merged**